fixed_pose:
  x: 0
  y: 0
  yaw: 0
last_good_pose:
  x: 70.85938262939453
  y: 15.6866569519043
  yaw: 0.1987975388765335